mapie/quantile_regression.py

```diff
@@ -453,6 +486,11 @@ def predict(  # type: ignore
             - [:, 1, :]: Upper bound of the prediction interval.
         """
         check_is_fitted(self, self.fit_attributes)
+        if alpha is not None:
+            raise ValueError(
+                "Alpha should not be specified in the prediction method"
+                + "with CQR"
```

Comment:
Don't use the acronym in error/warning messages as the message needs to be self-sufficient.

Reply:
done